ocfweb/docs/docs/staff/staffvm/ocfweb.md

```diff
@@ -0,0 +1,26 @@
+[[!meta title="Running ocfweb on your Staff VM"]]
+
+If you want to work on [`ocfweb`][0] on your Staff VM, it isn't as
+simple as cloning the repo and running `make dev`, unfortunately.
+In order to do so, you will need to add the `ocfweb` dev config to
+your Staff VM's Puppet configuration, so Puppet can install
+`ocfweb`'s dependencies. To do this, clone the [puppet repo][1] or
```

Comment:
Not sure what's happening on those, I don't think celery being uninstalled would be related to this, the dependencies I was thinking of were more like development headers to that things like compile correctly (like I believe the `cracklib` package needs some extra packages to be installed to run). Also I don't think redis should be needed, maybe a redis client or something? I thought that was installed with the requirements though python instead.

Yeah, or maybe `config files`, just to be more accurate anyway.